src/pages/compendia/[type].js

```diff
@@ -1,69 +1,80 @@
 import { useRouter } from 'next/router'
-import { useEffect, useState } from 'react'
-import { Box, Paragraph } from 'grommet'
-import { useCompendia } from 'hooks/useCompendia'
+import { Box, Tab } from 'grommet'
+import { useCompendiaContext } from 'hooks/useCompendiaContext'
 import { useResponsive } from 'hooks/useResponsive'
+import { api } from 'api'
+import { compendia as CompendiaConfig } from 'config'
 import getReadable from 'helpers/getReadable'
-import { FileDownload, Hero, Tabs } from 'components/Compendia'
-import { PageTitle } from 'components/shared/PageTitle'
 import { SignUpBlock } from 'components/shared/SignUpBlock'
-import { Spinner } from 'components/shared/Spinner'
+import { Tabs } from 'components/shared/Tabs'
+import { Hero } from 'components/Compendia/Hero'
+import { NormalizedTab } from 'components/Compendia/NormalizedTab'
+import { RNASeqTab } from 'components/Compendia/RNASeqTab'
 
-export const Compendia = () => {
+export const Compendia = ({ compendia, type }) => {
   const { setResponsive } = useResponsive()
-  const {
-    asPath,
-    query: { type: currentType }
-  } = useRouter()
-  const { hasError, loading, getCompendia } = useCompendia()
-  const [compendia, setCompendia] = useState(null)
-  const isLoading = loading || !compendia
-  const isDownload = asPath.includes('download')
-  const titlePrefix = `${
-    isDownload ? 'Download Compendia' : getReadable(currentType)
-  } -`
+  const { push } = useRouter()
+  // initializes the compendia contexts
+  useCompendiaContext(compendia, type)
 
-  // fetches both compendia on page load
-  useEffect(() => {
-    const fetchCompendia = async () => {
-      const [normalizedResponse, rnaSeqResponse] = await Promise.all([
-        getCompendia('normalized'),
-        getCompendia('rna-seq')
-      ])
-
-      setCompendia({
-        normalized: normalizedResponse,
-        'rna-seq': rnaSeqResponse
-      })
+  const tabConfigs = [
+    {
+      id: 'normalized',
+      Component: NormalizedTab
+    },
+    {
+      id: 'rna-seq',
+      Component: RNASeqTab
     }
-
-    fetchCompendia()
-  }, [])
+  ]
+  // sets the active tab index based on the selected type
+  const activeIndex = tabConfigs.findIndex(({ id }) => id === type)
```

Comment:
```suggestion
  const activeIndex = tabConfigs.findIndex((config) => config.type === type)
```

Maybe change id -> type and rewrite this way so you dont need the comment.

Reply:
I've renamed it to `tab` to avoid ESLint `no-unused-vars` warning and removed the comment 👍

Reply:
**UPDATE:**  I renamed `tabConfigs.tab` to `tabConfigs.type` and removed the destructuring when mapping the config in 691f095.